scripts/build.sh

```diff
@@ -45,6 +52,12 @@ cleanup_and_install() {
   if [ "${anchor}" = true ] ; then
     light-anchor build
   fi
+  
+
+  
+  if [ "${dir}" != "light-system-programs" ]; then
+      yarn link
+  fi
```

Comment:
why do we `yarn link` cli, light-circuits, and relayer?

Reply:
`yarn link` without arguments consumes the links in the current project. Doing `yarn link` in cli, applies the @lightprotocol/zk.js (and other links we made) in the cli project. Without it, cli would still try to use the release specified in package.json, so it would end up downloading it from npmjs.

Reply:
afaik that's incorrect. `yarn link` without arguments makes your package available to all other local projects to consume. the script uses `yarn link light-zk.ks` inside the consuming repos

Reply:
the script ordering may be confusing since yarn link happens at the end; the reason it works is that we're executing the scripts first with light-prover, then zk.js, and lastly the consuming projects

Reply:
The syntax yarn link @lightprotocol/zk.js and @lightprotocol/prover.js works for me.

Reply:
it's still not clear to me:
> why do we `yarn link` cli, light-circuits, and relayer?

Reply:
it's not necessary but it also doesn't do anything

Reply:
> afaik that's incorrect. `yarn link` without arguments makes your package available to all other local projects to consume. the script uses `yarn link light-zk.ks` inside the consuming repos

OK, you are right. Sorry!